src/vaults/legacy/LegacyVaultAdapter.sol

```diff
@@ -4,7 +4,7 @@ pragma solidity 0.8.28;
 import {PoolId} from "src/common/types/PoolId.sol";
 import {ShareClassId} from "src/common/types/ShareClassId.sol";
 
-import {IERC7540} from "src/vaults/legacy/interfaces/IERC7540.sol";
+import {IVault} from "src/vaults/legacy/interfaces/IVault.sol";
```

Comment:
Nit: Not sure if we want to keep IERC7540 here. If not, maybe `ILegacyVault` might be more appropriate?

Reply:
Agree with ILegacyVault name. I removed the IERC7540, because the interface adds poolId and scId, which are not IERC7540 compatible.

Reply:
Done!